site/src/testHelpers/handlers.ts

```diff
@@ -311,4 +311,11 @@ export const handlers = [
   rest.get("/api/v2/deployment/stats", (_, res, ctx) => {
     return res(ctx.status(200), ctx.json(M.MockDeploymentStats))
   }),
+
+  rest.get(
+    "/api/v2/workspacebuilds/:workspaceBuildId/parameters",
+    (_, res, ctx) => {
+      return res(ctx.status(200), ctx.json([M.MockTemplateVersionParameter1]))
```

Comment:
Something is fishy here...

`/api/v2/workspacebuilds/:workspaceBuildId/parameters` should return an array of `MockWorkspaceBuildParameter...`.

Reply:
Ahhhh, ok! Thank you.